packages/components/src/components/dropdown/helpers.js

```diff
@@ -1,9 +1,9 @@
-import { isArrayLike } from 'mobx';
+import PropTypes from 'prop-types';
 
 export const getDisplayText = (list, value) => {
```

Comment:
@reza-fs can we move the functions into our components or if we wanna keep it separated, don't use name helper?
you can use name `dropdown.js`. currently when you search in your editor it show a mil helper files.

Reply:
Done.